### DISCUSSION TITLE
How about some docs, examples and more info in the readme?

### DISCUSSION THREAD

**Discussion by Tony20221** [weblinq/WebLinq#149] [General]
How about some docs, examples and more info in the readme?

There's nothing on the home page that tells me what this repo does, how's it's used, examples, docs... anything! Instead of just one line.

**Comment by atifaziz** [comment]
Yes, I agree. Documentation and some introductory information are sorely missing, but I am afraid to say that's not going to change very soon. This project, while in use in production for many years and serving hundreds of scrapes, needs a lot of investment to polish for the general public consumption. That said, you'll find there are [quite some examples that show different use cases](https://github.com/atifaziz/WebLinq/blob/17e90cc06d027b19c0fec309c24e816c70de8f94/eg/Program.cs) and which can hopefully give you an idea of the value proposition and whether it's something you wish to invest into.
